cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -6,20 +6,28 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"fmt"
+	"math/big"
 	"os"
 	"os/signal"
 	"sync"
 	"syscall"
 	"time"
 
+	"github.com/Sifchain/sifnode/cmd/ebrelayer/txs"
+	"github.com/Sifchain/sifnode/cmd/ebrelayer/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common"
 	tmKv "github.com/tendermint/tendermint/libs/kv"
 	tmLog "github.com/tendermint/tendermint/libs/log"
 	tmClient "github.com/tendermint/tendermint/rpc/client/http"
 	tmTypes "github.com/tendermint/tendermint/types"
 
-	"github.com/Sifchain/sifnode/cmd/ebrelayer/txs"
-	"github.com/Sifchain/sifnode/cmd/ebrelayer/types"
+	ethbridge "github.com/Sifchain/sifnode/x/ethbridge/types"
+)
+
+const (
+	// RelayerNodeNumber defined for estimate total fee of Ethereum transaction
+	RelayerNodeNumber = 3
```

Comment:
Why is this hardcoded to 3?

Reply:
we never know how many validators online and get the exact total cost of transaction fee in Ethereum. so it is hardcoded for our testnet now.

Reply:
Doesn't the testnet have 4?

Reply:
As I know, three validators are active, one is backup.